src/openrct2-ui/windows/TileInspector.cpp

```diff
@@ -1832,7 +1832,7 @@ static void window_tile_inspector_paint(rct_window* w, rct_drawpixelinfo* dpi)
             {
                 // Details
                 // Path name
-                rct_string_id pathNameId = tileElement->AsPath()->GetPathEntry()->string_idx;
+                rct_string_id pathNameId = tileElement->AsPath()->GetSurfaceEntry()->string_idx;
```

Comment:
This doesn't look right. This is about getting the string ID for the path's name. Assuming it still gives back the right string, why was this renamed to `GetSurfaceEntry`?

Reply:
There are path surfaces and path railings. Calling the former just "path" is confusing in its own right.